pkg/nimble/contrib/controller_init.c

```diff
@@ -27,6 +27,12 @@
 #include "nrf_clock.h"
 #endif
 
+#ifdef DEVELHELP
+#define FLAGS           (THREAD_CREATE_STACKTEST)
+#else
+#define FLAGS           (0)
+#endif
```

Comment:
I don't think this distinction between compiling with and without `DEVELHELP` is a good idea. If I read `core/thread.c` correctly, `THREAD_CREATE_STACKTEST` is only used when either `DEVELHELP` or `SCHED_TEST_STACK` is enabled and is ignored otherwise. Changing the handling here to `DEVELHELP` only could cause confusion with people when stacktest is not enable while they might have enabled `SCHED_TEST_STACK`

Reply:
i was already thinking that we covered this thing globally somewhere, but I could not remember so hence this PR. Will adapt to get rid of the duplication...